README.md

```diff
@@ -5,40 +5,56 @@ Alpine Linux package builder for ROS (Robot Operating System)
 ## Build builder container
 
 ```shell
-docker build -t alpineros/ros-abuild:3.11-noetic .
+docker pull ghcr.io/alpine-ros/ros-abuild:3.11-noetic
 ```
-
-or pull from GitHub Container Registry
+or build locally
 
 ```shell
-docker pull ghcr.io/alpine-ros/ros-abuild:3.11-noetic
-docker tag ghcr.io/alpine-ros/ros-abuild:3.11-noetic alpineros/ros-abuild:3.11-noetic
+docker build -t ghcr.io/alpine-ros/ros-abuild:3.11-noetic .
 ```
 
 ## Build ROS package(s)
 
-In ROS package directory:
+### Just build and test
+
 ```shell
 docker run -it --rm \
-  -v $(pwd):/src/$(basename $(pwd)):ro \
-  alpineros/ros-abuild:3.11-noetic
+  -v $(pwd):/src:ro \
```

Comment:
[optional] it might be good to specify that `$(pwd)` is supposed to be the folder of the package to be built.